src/_product-updates/2024-04-05-love-cycle.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Love Cycle
+tags: [Improvement]
+image: /images/changelog/love-cycle.png
+---
+
+![new-dashboard-2024.jpg](/images/changelog/love-cycle.png)
+
+We have a tradition of scheduling what we call Love Cycles between our development cycles: times dedicated to improving Bump.sh, responding to specific user requests that we find relevant, and providing a general polish.
+
+Our last Love Cycle has recently ended, and we wanted to share with you all the small changes we've made. If you have a feature in mind or a need, don't hesitate to share your feedback with us at hello (at) bump.sh!
+
+Improvements:
+
+- Overlay Support: [Overlays](https://docs.bump.sh/help/specification-support/overlays/) are a new feature (beta) of OpenAPI that allow applying a layer over an API document without modifying it, to adapt to a specific context. Now supported by Bump.sh, feel free to read our [recent guide](https://docs.bump.sh/guides/openapi/augmenting-generated-openapi/) on the topic!
+- We added a [new command to our CLI](https://github.com/bump-sh/cli?tab=readme-ov-file#bump-overlay-definition_file-overlay_file) that allows applying an overlay to documentation.
+- Examples will now display correctly regardless of the chosen format. Bonus for those in XML which will also display the appropriate highlight.
+- [OpenAPI] Improved support for externalValue which now displays a link to the designated resource.
```

Comment:
I guess there the link to the OpenAPI specification page missing?

Reply:
If you only want a highlight, there's a ==markdown syntax for that== (see [our own help page about markdown](https://docs.bump.sh/help/enhance-documentation-content/markdown-support/#common-markdown-syntax-support))